src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -253,6 +253,13 @@ public void testRequiredTokenIsEmptyIntArray() throws Exception {
         }
     }
 
+    @Test
+    public void testBehaviourWithZeroChecks() throws Exception {
+        final TreeWalker treeWalker = new TreeWalker();
+        treeWalker.configure(new DefaultConfiguration("default config"));
+        treeWalker.processFiltered(temporaryFolder.newFile("file.java"), new ArrayList<>());
```

Comment:
@MEZk got it. Also, there are some other tests in TreeWalkerTest.java where nothing is asserted. We might need issues for that too perhaps ?